### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 language: node_js
 node_js:
-  - '5'
-  - '4'
-  - '0.10'
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
+  - "iojs"
```
Comment (kevva) [inline .travis.yml]: Isn't `0.12`, `4` and `5` enough?

Comment (XhmikosR) [inline .travis.yml]: This isn't my choice. It's what @shama wanted and we have in grunt-contrib-internal.

File: appveyor.yml [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
     - nodejs_version: "0.10"
+      platform: x86
+    - nodejs_version: "0.12"
+      platform: x86
+    - nodejs_version: "4"
+      platform: x64
+    - nodejs_version: "4"
+      platform: x86
+    - nodejs_version: "5"
+      platform: x86
```
Comment (kevva) [inline appveyor.yml]: This seems unrelated to a dependency bump.

Comment (XhmikosR) [inline appveyor.yml]: Have you even checked before commenting??

We normalize CI configs via grunt-contrib-internal and not per plugin.

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 before_install:
-  - npm install -g npm
+  - if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then npm install -g npm@2; fi
+matrix:
+  fast_finish: true
+cache:
+  directories:
+    - node_modules
```
Comment (kevva) [inline .travis.yml]: This could potentially lead to unnoticed broken builds when a subdependency breaks.

**Comment by kevva** [comment]
Oh, never mind. Saw your last comments :).

File: .travis.yml (hunk 3 above)
Comment (XhmikosR) [inline .travis.yml]: That is true but mostly for packages that use git as a package version.

Everyone from the team has the needed rights to clear the cache if needed so we'll see how it goes.

So far, I haven't seen any issue in any contrib plugin; only the gruntjs.com package, but that is using a package via git.
